include/aikido/util/CatkinResourceRetriever.hpp

```diff
@@ -1,9 +1,9 @@
 #ifndef AIKIDO_UTIL_CATKINRESOURCERETRIEVER_HPP_
 #define AIKIDO_UTIL_CATKINRESOURCERETRIEVER_HPP_
 #include <string>
-#include <unordered_map>
 #include <vector>
 #include <dart/common/ResourceRetriever.hpp>
+#include <unordered_map>
```

Comment:
I updated the related option. We can group and prioritize headers using `IncludeCategories` option with regex for each group. @mkoval could you take a look the regex again. I'm sure you know regex much better than me.